### DISCUSSION TITLE
Objective function

### DISCUSSION THREAD

**Discussion by Mostafa494** [Critical-Infrastructure-Systems-Lab/PowNet#4] [Q&A]
Dear Sir,
Thank you for your efforts.
Kindly guide me how to extract the objective function values so as to compare the results with other techniques, I have read the code and run all attached files, however the results do not include the obj function result.

Regards

**Comment by kamal0013** [comment]
Thanks! As I responded to the original question, the objective function is actually the hourly production costs. Please, check [this example script](https://github.com/kamal0013/PowNet/blob/master/Model_withdata/analysis_genmix_cost_emi.ipynb) to estimate production costs at hourly and annual scales.

**Reply by Mostafa494** [reply]
Thank you Dr. Kamal
Could I apply the system on IEEE 118 system?
How can I define the load demand curve in MW not in MWh as I do not have load data for each bus for each hour in the system?

**Reply by Mostafa494** [reply]
> Thanks! As I responded to the original question, the objective function is actually the hourly production costs. Please, check [this example script](https://github.com/kamal0013/PowNet/blob/master/Model_withdata/analysis_genmix_cost_emi.ipynb) to estimate production costs at hourly and annual scales.

Thank you Dr. Kamal for your support.
The script worked properly. Would you mind share with me the script for Thailand case so I could add renewable energies to the system.
Regards.

**Reply by kamal0013** [reply]
Sure, the Thailand model and data are available [here](https://github.com/kamal0013/PowNet-Thailand).

**Comment by Mostafa494** [comment]
Dear Dr. Kamal,
Thank you for your help.
Actually I do not have hourly load data for each node (bus), So Is there is a way that I could define a hourly load pattern as a total and not for each node ( as I have a lack of data in this area for excel sheet data_load).
Regards


Mostafa Hasaneen
Technical Studies Engineer
Energy Efficiency & Climate Change Department (EEACC)
The Egyptian Ministry of Electricity & Renewable Energy

________________________________
